don't download stemmer/re2
dynlink boost
todo: pre-download columnar
todo: look at icu again
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 48eff62..ac72c47 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -200,12 +200,12 @@ else ()
 endif ()
 
 # Check for RE2 build
-set (WITH_RE2_FORCE_STATIC 1 CACHE BOOL "force to compile re2 from sources")
-with_get (re2 "RE2" "a regular expression library")
+set (WITH_RE2_FORCE_STATIC 0 CACHE BOOL "force to compile re2 from sources")
+#with_get (re2 "RE2" "a regular expression library")
 
 # Check for Libstemmer build
-set (WITH_STEMMER_FORCE_STATIC 1 CACHE BOOL "force to compile stemmer from sources")
-with_get (stemmer "Stemmer" "stemming library (Snowball)")
+set (WITH_STEMMER_FORCE_STATIC 0 CACHE BOOL "force to compile stemmer from sources")
+#with_get (stemmer "Stemmer" "stemming library (Snowball)")
 
 # Check for ICU build
 set (WITH_ICU_FORCE_STATIC 1 CACHE BOOL "force to compile ICU from sources")
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 2956e7a..62037ed 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -100,7 +100,7 @@ endif()
 
 # Boost is required dependency now
 set (Boost_USE_MULTITHREADED ON)
-set (Boost_USE_STATIC_LIBS ON)
+set (Boost_USE_STATIC_LIBS OFF)
 # todo: as alternative we can try to recompile boost with clang, but precompiled also quite ok
 if (CMAKE_CROSSCOMPILING AND CMAKE_SYSTEM_NAME STREQUAL Windows)
 	set (Boost_COMPILER vc142)
